pkgs/development/lua-modules/overrides.nix

```diff
@@ -585,6 +586,15 @@ in
     };
   });
 
+  psl = prev.psl.overrideAttrs (drv: {
+    buildInputs = drv.buildInputs or [ ] ++ [ libpsl ];
+
+    luarocksConfig.variables = {
```

Comment:
this erases the generated luarocksConfig, do `luarocksConfig = drv.luarocksConfig // { variables = { .. };` or recursive update, look how it's done for other packages.

Reply:
👍🏼 changed